HTML/style.css

```diff
@@ -0,0 +1,56 @@
+.level1 {
+    padding: 10px;
+    background-color: #f0f0f0;
+}
+
+.level2 {
+    padding: 10px;
+    background-color: d0d0d0;
+}
+
+.level3 {
+    padding: 10px;
+    background-color: #b0b0b0;
+}
+
+.level4 {
+    padding: 10px;
+    background-color: #909090;
+}
+
+.level5 {
+    padding: 10px;
+    background-color: #707070;
+    color: white;
+}
+
+.first {
+    margin-left: 18%;
+    margin-right: auto;
+    background-color: #fff;
+    font-family: sans-serif;
+    font-size: large;
+  }
+  .first h1 {
+    font-size: 4em;
+  }
+  .first h1 .mundo {
+    text-decoration: line-through;
+  }
+  .first input {
+    background-color: #00F3BB;
+    border: F3BB;
+    height: 2.5emmm;
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/16#issuecomment-2211163237>#b402cc</a><span> - 07/05/2024, 04:55 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The value '2.5emmm' for the height property is invalid and will likely cause rendering issues. This can lead to performance problems as the browser may need to reflow and repaint the layout multiple times to handle the error.
</div>
<div id="fix">
<b>Fix:</b> Correct the height value to a valid CSS unit, such as '2.5em'.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -43,7 +43,7 @@
  .first input {
      background-color: #00F3BB;
      border: F3BB;
 -    height: 2.5emmm;
 +    height: 2.5em;
      margin-top: 3;
      padding: 1%;
    }
```
</div>
</details>
</div>
<br>


</div>